internal/operators/ocs/consts.go

```diff
@@ -3,10 +3,14 @@ package ocs
 type ocsDeploymentMode string
 
 const (
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	minDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	CPUCompactMode      int64             = 6  // per host requires 6 cpus for OCS in compact mode
+	CPUMinimalMode      int64             = 4  // per host requires at least 4 cpus for OCS in minimal mode
```

Comment:
are 12 CPus enough for a cluster in minimal mode?

Reply:
Yes, for minimal mode we need 18 CPU (excluding OCP), and each disk requires 2 CPU so 12 CPU excluding the disks

Reply:
In minimal mode, each OSD uses 1 CPU

Reply:
For minimal mode, we need 8 cpu per node ( including OCP and disk req. ), so thought it would be safe to consider 2 cpu for OSD, hence excluding OCP and disk req. ( 8-2-2=4).  
Let me know WDYT.